examples/README.md

```diff
@@ -41,7 +41,7 @@ This directory contains example using the [`bokeh.plotting`](http://bokeh.pydata
 
 * `notebook` examples that display inline in [Jupyter](http://jupyter.org) notebooks
 
-* `server` examples that send output to a Bokeh server, using the python client library for the server 
+* `server` examples that send output to a Bokeh server, using the python client library for the server
```

Comment:
charts also has this same seperation - should we have this list higher up the readme.